azure-iot-device/azure/iot/device/iothub/pipeline/mqtt_pipeline.py

```diff
@@ -593,11 +594,11 @@ def pipeline_configuration(self):
         Pipeline Configuration for the pipeline. Note that while a new config object cannot be
         provided (read-only), the values stored in the config object CAN be changed.
         """
-        return self._pipeline.pipeline_configuration
+        return self.nucleus.pipeline_configuration
 
     @property
     def connected(self):
         """
         Read-only property to indicate if the transport is connected or not.
         """
-        return self._pipeline.connected
+        return self.nucleus.connected
```

Comment:
>nucleus [](http://example.com/codeflow?start=20&length=7)

consistency: sometimes it's self.pipeline_nucleus, other times it's self.nucleus. I personally like the shorter term, but I like consistency even more.